upstream/solr6/local/92-init-solr.sh

```diff
@@ -152,10 +152,10 @@ function makeConfigs {
         setOption 'solr.suggester.enabled' "$ALFRESCO_SOLR_SUGGESTER_ENABLED" "$CONFIG_FILE_CORE"
         # SOLR_BACKUP_DIR only useful for ASS v2.0.2* and up
         if [ $coreAlfrescoName != $coreName ]; then
```

Comment:
This will be a breaking change for solr versions v2.2 <-> v2.5.
You will need to make it conditional based on the minor version

Reply:
no need since we don't build them anymore